README.md

```diff
@@ -73,3 +73,20 @@ docker run -it --rm -p 8080:8080 example-spring-boot
 # In another terminal
 curl localhost:8080
 ```
+
+## Checkpoint as Dockerfile build step
+
+In order to perform a checkpoint in a container we need those extra capabilites mentioned in the commands above. Regular `docker build ...` does not include these and therefore it is not possible to do the checkpoint as a build step - unless we create a docker buildx builder that will include these. Note that you require a recent version of Docker BuildKit to do so.
+
+```
+docker buildx create --buildkitd-flags '--allow-insecure-entitlement security.insecure' --name privileged-builder
+docker buildx build --load --builder privileged-builder --allow=security.insecure -f Dockerfile.privileged -t example-spring-boot .
```

Comment:
This is unfortunate, but it looks the best think available according to ﻿https://docs.docker.com/reference/dockerfile/#run---security.

Could you add the link to the doc -- in the case it's updated with something more convenient in the future.

Ideally, GHA can run the new flow, to verify it works.

And it seems GHA is not enabled on PRs :(

Reply:
Docker docs link added.

Wrt GHA it doesn't run the other containers either (yes, I've added those instructions and Dockerfiles). We might handle it in another PR if you enable PR actions.